docs/source/bricks.rst

```diff
@@ -750,3 +750,18 @@ files to an S3 bucket.
 
   upload_staged_files()
 
+``stage_for_datasaur``
+--------------------------
+Formats a list of ``Text`` elements as input to token based tasks in Datasaur.
+
+Example: 
+
+.. code:: python
+
+  from unstructured.staging.datasaur import stage_for_datasaur
+  input  = [Text("Text1"),Text("Text2")]
+  output = stage_for_datasaur(input)
```

Comment:
Can you change `output` to `datasaur_data`?

Reply:
Yeah, of course